quiz.rb

```diff
@@ -1,20 +1,68 @@
+
 # Question 1
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
+class Dog
+
+  def initialize(name,age,breed)
+    @name = name
+    @age = age
+    @breed = breed
+  end
+
+  attr_reader :age
```

Comment:
One tip: you can actually write all of these in a single line, separating each symbol with commas.
e.g.
`attr_reader :name, :breed, :age`